wiki/Contests/Skinning_Contest/5/en.md

```diff
@@ -0,0 +1,78 @@
+---
+tags:
+  - skinning
+  - skinship
+  - skins
+---
+
+# Skinning Contest #5
```

Comment:
```suggestion
# Skinning Contest #5: Unchained
```

Reply:
seems like the previous ones don't include the subtitle for some reason

Reply:
Have applied it to this contest and all past ones